admin/controllers/tech-support.js

```diff
@@ -492,11 +493,11 @@ const controller = {
   },
 
   postCheckResultsResyncAll: async function postCheckResultsResyncAll (req, res, next) {
-    res.locals.pageTitle = 'Resync All Results'
+    res.locals.pageTitle = 'Synchronise new checks from Table Storage to main SQL database'
     const resyncAll = req.body.resyncAll === 'true' // convert string to bool- the user must tick the checkbox
     try {
       await resultsResyncService.resyncAllChecks(resyncAll)
-      req.breadcrumbs('Check Results - Resync All')
+      req.breadcrumbs('Check Results - Resync Uncalculated')
```

Comment:
I believe the "All" is correct here - it will resynchronise checks that are already in the db from table storage.